src/store/modules/Settings/NetworkStore.js

```diff
@@ -360,7 +374,11 @@ const NetworkStore = {
           `/redfish/v1/Managers/bmc/EthernetInterfaces/${state.selectedInterfaceId}`,
           updatedIpv6Array
         )
-        .then(dispatch('getEthernetData'))
+        .then(() => {
+          setTimeout(() => {
```

Comment:
setTimeout is repeated multiple times, can this be reused by making into one single method

Reply:
Done